gr-digital/lib/correlate_access_code_bb_impl.cc

```diff
@@ -98,7 +98,7 @@ int correlate_access_code_bb_impl::work(int noutput_items,
 
         // compute hamming distance between desired access code and current data
         unsigned long long wrong_bits = 0;
-        unsigned int nwrong = d_threshold + 1;
+        unsigned int nwrong;
```

Comment:
Since this is C++, how about setting each variable as it is being used, instead of beforehand? That would simplify the code as well as fix little issues like this one you found (which is valid, BTW).

Reply:
Meaning, do something like:
```
unsigned long long wrong_bits = (d_data_reg ^ d_access_code) & d_mask;
unsigned int nwrong = gr::blocks::count_bits64(wrong_bits);
...
bool new_flag = (nwrong <= d_threshold);
...
```
and change `new_flag` to a bool type, since that's what it actually is (and, again, this is C++ ..)